DataLayer/Migrations/Scripts/GameEvent_Up.sql

```diff
@@ -0,0 +1,74 @@
+﻿DROP TRIGGER IF EXISTS TR_User_InsteadOfDelete;
+
+GO
+
+CREATE TRIGGER TR_User_InsteadOfDelete
+  ON [User]
+  INSTEAD OF DELETE
+AS
+  BEGIN
+    DELETE FROM Friendship
+    FROM DELETED
+    WHERE RequestedById = DELETED.Id
+       OR RequestedToId = DELETED.Id;
+
+	DELETE FROM GameEventParticipation
+	FROM DELETED
+	WHERE GameEventParticipation.ParticipantId = DELETED.Id;
```

Comment:
Fixed. You meant that "OR", didn't you?

Reply:
No, I didn't, I meant lines 15 through 17. I thought they were misindented cause that what it looked like, but it seems they're indented with tabs instead of spaces. Which is bad. The `OR` was OK, so bring it back :)

Reply:
Replaced all tabs with spaces.